src/devices/Common/Iot/Device/Common/WeatherHelper.cs

```diff
@@ -331,5 +331,84 @@ public static Length CalculateAltitude(Pressure pressure, Pressure seaLevelPress
         }
 
         #endregion
+
+        /// <summary>
+        /// Simplified air density (not taking humidity into account)
+        /// </summary>
+        /// <param name="airPressure">Measured air pressure</param>
+        /// <param name="temperature">Measured temperature</param>
+        /// <returns>Approximate standard air density</returns>
+        public static Density CalculateAirDensity(Pressure airPressure, Temperature temperature)
+        {
+            double gasConstant = 287.058; // J / (kg * K), for dry air
```

Comment:
I assume that for all the constants used here, the same source for them apply than all the existing calculation? Otherwise, would be great to add any additional source.

Reply:
Yes, this is basically the same as below, when humidity is not known.

Reply:
can we extract heard-coded repeating numbers to constants in this file?

Reply:
@krwq We can, but I think this would only be useful for numbers that occur more than once. Otherwise it doesn't improve readability IMHO.

Reply:
I can see the same constant used here and method below

Reply:
ah, I see you changed that, thanks!